README.md

```diff
@@ -0,0 +1,138 @@
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Covid] <a name="about-project"></a>
+
+**Covid** is a App that provides reports of Covid-19. It displays data of different countries and histrories of analysis made. 
+
+## 🛠 Built With<a name="built-with"></a>
+
+React
+React-Redux
+
+### Tech Stack <a name="tech-stack"></a>
+
+Fullstack developer: includes both front end and Backend developer.
+
+### Key Features <a name="key-features"></a>
+
+- **Displays data of Different Countries**
+- **Search feature**
+- **Details Page**
+
+- **[Linters]**
+- **[Node]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://644c0f92b8e2ca03c7137404--astonishing-figolla-eb6a4d.netlify.app/)
+- [Live Presentation Link] (https://www.loom.com/share/aaedfa07cc434d15b927bc67e7984218)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+https://github.com/Luchy-B/covid
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+https://github.com/Luchy-B/covid
+```
+
+### Install
+install this project with:
+
+```sh
+npm install
+```
+
+### Prerequisites
+
+In order to run this project you need:
+
+- install npm, node
+- Install code editor
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+npm start
+```
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+npm run test
+```
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Luchy-B)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/blessingokpara)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add search**
+- [ ] **Add details page**
+- [ ] **Add styling**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Luchy-B/covid/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project, kindly show your support by leaving a comment on how to improve.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank Microverse for Guiding me through this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] You should give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).